src/Microsoft.ML.Core/Utilities/Contracts.cs

```diff
@@ -737,7 +737,11 @@ public static void CheckIO(this IExceptionContext ctx, bool f, string msg)
             if (!f)
                 throw ExceptIO(ctx, msg);
         }
-
+        public static void CheckIO(this IExceptionContext ctx, bool f, string msg, params object[] args)
+        {
+            if (!f)
```

Comment:
Is the convention to have single letter variables? Seems like golang's style :)

Reply:
I am not sure about the convention here, but we are using it for all the contract checks. I think we should keep them same for this pr. we can later modify if we want to